tools/src/main/java/org/wildfly/swarm/tools/exec/SwarmProcess.java

```diff
@@ -119,7 +119,9 @@ public int stop(long timeout, TimeUnit timeUnit) throws InterruptedException {
         }
         this.process.destroy();
         if (!this.process.waitFor(timeout, timeUnit)) {
-            process.destroyForcibly();
+            // Attempt to terminate the process forcibly and also wait for the same amount of time as the base attempt.
+            // This should potentially take care of the issues with the process not terminating on Windows CI build.
+            process.destroyForcibly().waitFor(timeout, timeUnit);
```

Comment:
Looking at the errors in the Windows build, looks like the process isn't terminating in 2 minutes, which in itself is weird and needs to be looked at. When invoking the "destroyForcibly()" method, we still need to check if the process is alive or not. Instead of adding a lot of logic, I am including an additional `waitFor` using the same arguments as the above invocation. Let's see if the Windows build goes through fine or not and how much longer does it take for completion.